security/cores.py

```diff
@@ -78,6 +80,26 @@ class InputRequestsLogISCore(RequestsLogISCore):
         (_('Extra information'), {'fields': ('slug', 'response_time', 'output_logged_requests')}),
     )
 
+    def get_form_fieldsets(self, request, obj=None):
+        form_fieldsets = list(super().get_form_fieldsets(request, obj))
+
+        app_names = {app.name for app in apps.get_app_configs()}
+
+        if 'security.contrib.reversion_log' in app_names and obj and obj.input_logged_request_revisions.exists():
+            from .contrib.reversion_log.views import RequestRevisionTabularInlineFormView
+
+            form_fieldsets.append((_('Revisions'), {'inline_view': RequestRevisionTabularInlineFormView}))
+
+        if (settings.SHOW_DEBUG_TOOLBAR and 'security.contrib.debug_toolbar_log' in app_names
+                and obj and hasattr(obj, 'input_logged_request_toolbar')):
+            form_fieldsets.append((None, {'fields': ('debug_toolbar',)}))
```

Comment:
This is a little hack :D isn't it? You are trying to display it as normal field but it will render as sidebar panel, right?

Reply:
Yes exactly